### DISCUSSION TITLE
did I need to do or  review the fr translation ?

### DISCUSSION THREAD

**Discussion by Moosh-be** [bluecaret/carettab#352] [Ideas and Feature Requests]
Is there some big changes  in  V4 about locale ?

**Comment by bluecaret** [comment]
There were a lot of changes to the phrasing of things between v3 and v4, and since I only know English myself, I had to start over with all translations using a machine translation, which is certainly not perfect.

If you'd like to contribute to the translations, I'd accept a pull request for any corrections that are needed. 

Thank you.

**Comment by Moosh-be** [comment]
PErfect. I do it and I  create a  PR ;)
